insights/parsers/openstack_server_list_all.py

```diff
@@ -0,0 +1,64 @@
+"""
+openstack server list --all command - Command
+=============================================
+Parsing and extracting data from output of command ``/usr/bin/openstack server list --all``.
+--------------------------------------------------------------------------------------------
+"""
+
+from insights import CommandParser, parser, LegacyItemAccess
+from insights.parsers import ParseException, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.openstack_server_list_all)
+class OpenstackServerListAll(LegacyItemAccess, CommandParser):
+    """
+    The output of command ``/usr/bin/openstack server list --all`` is in table format,
+    contains 'Variable_name' and 'Value' two columns.
+    This parser will parse the table and set each variable as an class
+    attribute. The unparsable lines are stored in the ``bad_lines`` property list.
+    Sample command output::
+
+        +--------------------------------------+--------------+--------+------------------------+----------------+------------+
+        | ID                                   | Name         | Status | Networks               | Image          | Flavor     |
+        +--------------------------------------+--------------+--------+------------------------+----------------+------------+
+        | 410b05bb-59b7-4b4c-88e9-975c811d68da | compute-0    | ACTIVE | ctlplane=192.168.24.20 | overcloud-full | compute    |
+        | f891e98b-4df6-4c90-9bf1-39cf8ac900b0 | compute-1    | ACTIVE | ctlplane=192.168.24.9  | overcloud-full | compute    |
+        | 3d62cd7e-41d2-43dd-a5bf-5935bc319fae | controller-0 | ACTIVE | ctlplane=192.168.24.10 | overcloud-full | controller |
+        +--------------------------------------+--------------+--------+------------------------+----------------+------------+
+    """
```

Comment:
Please add **Attributes:** and **Examples:** sections to your docstring to help developers understand how to use your parser.  See [this reference](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google) and the [insights-core documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for example docstrings.